extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -95,6 +95,7 @@ describes.realWin(
       resetServiceForTesting(win, 'consentStateManager');
       registerServiceBuilder(win, 'consentStateManager', function () {
         return Promise.resolve({
+          consentPageViewID64: () => 'IF_sLhxPvk0-JaagInsH8A',
```

Comment:
for readability keep this a simple string like `consent_page_view_id_64`.

Reply:
Good idea. Thanks